### DISCUSSION TITLE
Verify email

### DISCUSSION THREAD

**Discussion by omeruren** [community/community#133722] [GitHub Education]
### Hi everyone,

I'm getting error while verifying my student e-mail .

I'm recieving the verify e-mail but i'cant verify my e-mail.
I'm getting error saying : "There was an error verifying your email."

**Comment by akibhaider** [comment]
-> Double-Check your e-mail: Make sure you're entering the correct email address and that it matches the one you registered with GitHub.
-> Use the Latest Verification Link: If you've received multiple verification emails, use the link from the most recent one.
-> Check for Typos: Ensure that there are no typos in the email address you're trying to verify. (A blank space at the end is the most common one)
-> Clear Browser Cache and Cookies: Sometimes, browser issues can cause problems. Try clearing your browser's cache and cookies, or use a different browser or incognito mode.
-> Check for Issues on GitHub's End: The issue might be on GitHub's end. Check GitHub's status page: (https://www.githubstatus.com/) to see if there are any ongoing issues.
-> Try Again Later: If the issue persists, try again later. Could be a temporary issue.
